example-app/santoku-app-backend/src/main/java/jp/fintan/mobile/santokuapp/infrastructure/persistence/entity/AccountDeviceTokenEntity.java

```diff
@@ -0,0 +1,44 @@
+package jp.fintan.mobile.santokuapp.infrastructure.persistence.entity;
+
+import java.sql.Timestamp;
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "account_device_token")
+@Access(AccessType.FIELD)
+public class AccountDeviceTokenEntity {
+
+  @Id private String accountId;
```

Comment:
アノテーションは一律上に定義だと違和感ありますでしょうか？

```suggestion
  @Id
  private String accountId;
```

Reply:
Google Java Formatで自動フォーマットかけたせい？なのか、こうなりますね